transformation/transformationPlugins/jsonMessageTransformers/jsonTypeMappingsSanitizationTransformer/README.md

```diff
@@ -0,0 +1,152 @@
+This transformer converts routes for various requests (see below) to indices that used
```

Comment:
The templates are in place to handle multi-type mappings.  I'm working on debugging through all of the bugs with that since the create index call's specification has changed over time.